docs/grants_proposal_discussion.md

```diff
@@ -313,7 +313,7 @@ https://radicle.community/t/radicle-2021-strategy-objectives/2424#high-level-obj
 
 Radicle is currently run by a couple dozen core members.
```

Comment:
Are you referring to Radicle code collab or radworks the dao here? If the dao, I would adjust wording to say "Radworks currently has a couple dozen core contributors across four Orgs" (not run by them) or Radworks is stewarded by a couple dozen core contributors across four Orgs and the driven by its token holders."